stor/tests/test_dx.py

```diff
@@ -868,7 +868,9 @@ def test_on_file_named_timed(self):
         url = urllib.request.urlopen(result)
         self.assertIn('attachment', url.headers['content-disposition'])
         self.assertIn('random.txt', url.headers['content-disposition'])
-        time.sleep(2)  # for link to expire
+        should_mock = self.cassette.rewound if hasattr(self, 'cassette') else False
```

Comment:
just set `cassette = None` at classlevel , then can say `if self.cassette and self.cassette.rewound`